layouts/partials/contribute.html

```diff
@@ -0,0 +1,19 @@
+<aside id="contribute">
+  <div>
+    <h4>Contribute:</h4>
+    <ul>
+      <li>
+        <a href="https://github.com/sensu/sensu-docs/edit/master/content/{{ .Page.File.Path | urlize }}" target="_blank">
+          <i class="fa fa-pencil-square-o" aria-hidden="true"></i>
+          Edit this page
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/sensu/sensu-docs/issues/new/choose" target="_blank">
+          <i class="fa fa-bug" aria-hidden="true"></i>
+          Report a docs issue
+        </a>
+      </li>
+    </ul>
```

Comment:
```suggestion
      <li>
        <a href="https://github.com/sensu/sensu-docs/blob/master/CONTRIBUTING.md" target="_blank">
          <i class="fa fa-info-circle" aria-hidden="true"></i>
          Read the contributing guide
        </a>
      </li>
      <li>
        <iframe src="https://ghbtns.com/github-btn.html?user=sensu&repo=sensu-docs&type=star&count=true" frameborder="0" scrolling="0" width="170" height="20"></iframe>
      </li>
    </ul>
```

Reply:
Just pulling the contributing guide and GH star button up into the new sidebar item too.